website/docs/docs/deploy/continuous-integration.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Continuous integration in dbt Cloud"
+sidebar_label: "Continuous integration"
+description: "You can set up Slim continuous integration (CI) checks to test every single change prior to deploying the code to production just like in a software development workflow."
+---
+
+To implement a continuous integration (CI) workflow in dbt Cloud, you can set up automation that tests code changes by running [Slim CI jobs](/docs/deploy/slim-ci-jobs) before merging to production. dbt Cloud tracks the state of what’s running in your production environment so, when you run a Slim CI job, only the modified data assets in your pull request (PR) and their downstream dependencies are built and tested in a staging schema. You can also view the status of the CI checks (tests) directly from within the PR; this information is posted to your Git provider as soon as a Slim CI job completes. Additionally, the teams in your organization can enable settings with your Git provider that allow PRs only with successful CI checks be approved for merging.  
```

Comment:
> Additionally, the teams in your organization can enable settings with your Git provider that allow PRs only with successful CI checks be approved for merging.  

This feel like something that we should be a little specific about. Is it a repo setting? If so "teams" might be a little misleading.

Reply:
i'm not quite sure i understand. curious, what specifics are you thinking about?